test/mocha/01-symmetric-key.js

```diff
@@ -29,14 +33,161 @@ describe('symmetric keys', () => {
           'https://w3id.org/security/suites/aes-2019/v1'
         ]);
       });
-    }); // end generateKey API
+      it('throws a DuplicateError when generating the same key twice',
+        async () => {
+          // keyId will be generated at the bedrock-kms-http layer
+          const keyId = `https://example.com/kms/${await generateId()}`;
```

Comment:
This is confusing ... won't this generate a new key ID? Was the goal to make it use the one from the `before` hook? Is that why the `before` hook was added? If so, I recommend removing the `before` hook and just having this test run the `generateKey` function twice -- the second time with the same locally generated key ID. That way the test is independent and can be `.only`'d, etc.